k8s/helmfile/env/local/api.values.yaml.gotmpl

```diff
@@ -61,3 +61,9 @@ storage:
 
 trustedProxy:
   proxies: ['*']
+
+queue:
+  queueNames: ['default']
```

Comment:
No, we're now moving it to be handled by horizon instead of having K8s create one deployment per queue name.

Reply:
so why isn't this then just `[]`?

Reply:
to answer my own question because then we'd end up with no horizon queue workers due to: https://github.com/wbstack/charts/blame/fed655db7a7167c61eba2c78905d0f6a679267a5/charts/api/templates/deployment-queue.yaml#L1

Reply:
Because I wanted this chart update to support both the "old mode" and Horizon, i.e. if this would be `[]`, we'd not deploy any queue worker deployment. Bad idea? I can see it's confusing, but thought it might be better to remove this at a later point when we know the new setup works well for us. If it doesn't this would allow us to roll back to the old state quickly.